source/macos/AdaptiveCards/AdaptiveCards/Views/ACRDateField.swift

```diff
@@ -112,18 +117,18 @@ class ACRDateField: NSView, InputHandlingViewProtocol {
     }
     
     var isValid: Bool {
-        return true
+        // TODO: Add min and max date checks too
+        return isBasicValidationsSatisfied
```

Comment:
Should we add min and max check here and move it away from UI?

Reply:
Nope not really. Just as a extra safety net.
Ideally that condition would never be met.